mmv1/products/containerattached/Cluster.yaml

```diff
@@ -337,3 +337,26 @@ properties:
         values:
           - :DISABLED
           - :PROJECT_SINGLETON_POLICY_ENFORCE
+  - !ruby/object:Api::Type::NestedObject
+    name: proxyConfig
+    required: false
+    description: |
+      Support for proxy configuration.
+    allow_empty_object: true
```

Comment:
For `proxyConfig` and `kubernetesSecret`, you may want to consider if you need `allow_empty_object: true`. It may not make sense to explicitly specify either of them as empty config objects, unless perhaps that is useful for clearing values. I see it on most of the other fields in this resource, so maybe easiest to just stay consistent.

Reply:
Yes, removed this since we don't consume empty objects for updates.